src/extension/helper_process.c

```diff
@@ -210,6 +217,45 @@ dd_conn *nullable dd_helper_mgr_cur_conn(void)
     return NULL;
 }
 
+static char *_concat_paths(const char *nonnull base, size_t base_len,
+    const char *nonnull file, size_t file_len)
+{
+    unsigned add_slash = base[base_len - 1] != '/';
+
+    // Since file comes from a literal, assume it has a safe length
+    char *ret = safe_pemalloc(base_len, 1, add_slash + file_len + 1, 1);
```

Comment:
I'd prefer `add_slash ? 1 : 0` and a bool add_slash to avoid a double-take here.